chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationStatus.java

```diff
@@ -16,9 +16,9 @@
 	
 	private String name;
 	
-	public CertificationStatus(Map<String, Object> certStatus){
-		this.id = Long.valueOf(certStatus.get("id").toString());
-		this.name = certStatus.get("name").toString();
+	public CertificationStatus(String name){
+		this.id = id;
+		this.name = name;
```

Comment:
Why is this file changing? I'm concerned that changing domain objects will have far-reaching effects. As well, there's no `id` that `this.id` can be set to in this method

Reply:
I created this domain object for the ICSFamilyTreeNode object, its not used anywhere else. I will get rid of the id though as that's not being used anymore.